waiter/src/waiter/util/utils.clj

```diff
@@ -120,6 +121,7 @@
       (instance? PersistentQueue v) (vec v)
       (instance? ManyToManyChannel v) (str v)
       (instance? Process v) (str v)
+      (instance? Timer$Context v) (str v)
```

Comment:
Maybe we should avoid rendering the timer context in the json?

Reply:
We definitely *can* do that—but this seems like a perfect use case for meta data.